ci-operator/config/openshift/origin/openshift-origin-release-4.18.yaml

```diff
@@ -649,6 +649,19 @@ tests:
       - observers-resource-watch
     workflow: openshift-e2e-external-aws
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-external-vsphere-ccm
+  optional: true
+  steps:
+    cluster_profile: vsphere-elastic
+    env:
+      TEST_SKIPS: provisioning should mount multiple PV pointing to the same storage
+        on the same node
```

Comment:
Don't you need to set the var `PATCH_INFRA_MANIFEST` to "enable" platform external?
Looks like it's disabled in the workflow https://github.com/openshift/release/blob/master/ci-operator/step-registry/upi/vsphere/platform-external/ccm/upi-vsphere-platform-external-ccm-workflow.yaml#L11

Reply:
nvm, this flag is not necessarily as installer will create correct manifests since 4.14+